HoloPipelines/README.md

```diff
@@ -114,7 +119,23 @@ pytest --cov
 
 
 ## API specification
-> TODO: Please add this section.
+> TODO: update this
+```
+GET /pipelineapp
+```
+To get a list of all available pipelines
+```
+GET /job/<jobid>/status
+```
+To get the status of a job with specific ID
+```
+POST /job
+```
+To start a job
+```
+POST /status
```

Comment:
Hmmm maybe this should be `POST /job/<jobid>/status` for coherence? However I am not sure we will have this endpoint? Let's discuss tomorrow.